frontend/src/components/Button/Button.tsx

```diff
@@ -47,7 +49,9 @@ export function Button({
         if (onClick) {
           onClick();
         }
-        e.stopPropagation();
+        if (!propagate) {
+          e.stopPropagation();
+        }
```

Comment:
Maybe I missed smth but what for you need `stopPropagation` here and what for you don't need it?

Reply:
So, I removed it and looks like all works fine

Reply:
I remembered: without it if we stop click propagation underlying `<Link>` not passes `state: { projectId, compoundId: p.compoundId, targetId: p.targetId },`